ui/src/literature/components/ParentRecordLinked.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import { Link } from 'react-router-dom';
+import { getRecordIdFromRef } from '../../common/utils';
+import { LITERATURE } from '../../common/routes';
+
+class ParentRecordLinked extends Component {
+  render() {
+    const { parentRecord } = this.props;
+    return (
+      (parentRecord && (
```

Comment:
better to move this empty check to the detail page.

Reply:
:+1: